tests/tsconfig.json

```diff
@@ -0,0 +1,14 @@
+{
+  "compilerOptions": {
+    "target": "es5",
+    "lib": ["es5", "dom", "es2016.array.include"],
+    "types": ["cypress", "cypress-file-upload", "node"],
+    "allowJs": true,
+    "isolatedModules": false,
+    "baseUrl": "./",
+    "noEmit": true,
+    "skipLibCheck": true
+  },
+  "include": ["**/*.ts"],
+  "exclude": ["node_modules"]
```

Comment:
Does this setup corresponds with other projects? (not only Konto)

Reply:
Currently I've push the basic tsconfig, which is needed to avoid errors. The original tsconfig contained things for the future that were needed in the Konto. I assumed that the full tech stack on both projects is 1:1. That's why I hadn't considered that it wouldn't be necessary.